### PR TITLE
Dislpay the actual filename (as is said in the message) not the description

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

**Comment by diba1013** [comment]
Changelog, as this affects review and recheck.cli?

File: src/main/java/de/retest/recheck/suite/flow/ApplyChangesToStatesFlow.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ private ApplyChangesToStatesFlow( final Persistence<SutState> persistence ) {
 				return Collections.emptyList();
 			}
 			goldenMasterProvider.saveGoldenMaster( file, newState );
-			return Collections.singletonList( actionChangeSet.getDescription() );
+			return Collections.singletonList( file.getName() );
```
Comment (diba1013) [inline src/main/java/de/retest/recheck/suite/flow/ApplyChangesToStatesFlow.java:76]: Do we want to transform this to a absolute path?

**Review by diba1013 — COMMENTED** [review]
Changelog, as this affects review and recheck.cli?

File: src/main/java/de/retest/recheck/suite/flow/ApplyChangesToStatesFlow.java (hunk 1 above)
Comment (roesslerj) [reply src/main/java/de/retest/recheck/suite/flow/ApplyChangesToStatesFlow.java:76]: I'd say we go with this for now and see how to further improve it later...

**Comment by roesslerj** [comment]
CHANGELOG adapted

**Review by diba1013 — APPROVED** [review]

**Pull request merged**